FBSDKCoreKit/FBSDKCoreKit/FBSDKAppEvents.m

```diff
@@ -523,8 +523,10 @@ + (void)updateUserProperties:(NSDictionary *)properties handler:(FBSDKGraphReque
     }
     return;
   }
+
+  NSString *appID = [FBSDKAppEvents loggingOverrideAppID] ?: [FBSDKSettings appID];
   NSDictionary *params = @{ @"data" : dataJSONString };
-  FBSDKGraphRequest *request = [[FBSDKGraphRequest alloc] initWithGraphPath:[NSString stringWithFormat:@"%@/user_properties", [FBSDKSettings appID]]
+  FBSDKGraphRequest *request = [[FBSDKGraphRequest alloc] initWithGraphPath:[NSString stringWithFormat:@"%@/user_properties", appID]
```

Comment:
I don't see a strong reason for changing this into a class method. It's fine to access the singleton like L973. 
`NSString *graphPath = [NSString stringWithFormat:@"%@/custom_audience_third_party_id", [[self singleton] appID]];`